packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -31,6 +31,7 @@ const {
   PotraceType,
   ImageFitType,
 } = require(`./types`)
+const { report } = require("process")
```

Comment:
```suggestion
```

Seems like a stray import. You'll use the passed in `reporter` for the call :)

Reply:
Oops, yeah. Thanks!